Makefile

```diff
@@ -14,3 +14,8 @@ lint-tests:
 
 unit-tests:
 	pytest --cache-clear --show-capture=stderr -vv
+
+.PHONY: installdeps
+installdeps:
+	pip install --upgrade pip
+	pip install -r test-requirements.txt
```

Comment:
i think this will fail to install the library, if that was the intention.  since the library is not mentioned in requirements.txt

Reply:
Right - I think I'm getting myself confused with the different ways things can be installed.

Is this what we want for each of these?
- `make installdeps` - install everything, library, base requirements, test requirements, complete requirements
- `pip install -r requirements.txt` - install only base requirements, but not library (like featuretools)
- `pip install -r test-requirements.txt` - install only additional test and complete dependencies (similar to featuretools)
- `pip install -r complete-requirement.txt` - install only complete dependencies

Reply:
yeah that sums it up

Reply:
Ok, I think the last commit should implement that behavior